example/remote/readme.md

```diff
@@ -0,0 +1,11 @@
+# Remote Communication with External Simulators
+
+This example shows
+1. setting up a simulation that uses an external tool to source spikes.
+2. the in-band control and spike exchange protocols. 
+3. use of the supplemental tools exposed by Arbor to handle these connections.
+
+Please run it with at least two MPI processes.
+
+Also included is a cross language client (Python)/ server (C++) example for
+demonstration. (Not yet tested due to OpenMPI fickleness)
```

Comment:
can you add a line on how you'd use the server/client example?

Reply:
Sure!